### PR TITLE
feat: add Scope of work to SRS

### PR DESCRIPTION
**Pull request by MohammadMohsinKhan**
# GitHub Ticket / Description

[SS-25](https://github.com/lilweege/SyntaxSentinels/issues/25)

## Type && Checklist

- [x] **New feature (new APIs, new functional methods)**
  - [ ] Created (or updated) unit test cases (if required)
  - [ ] Passed unit testing for **ALL** cases
  - [ ] Created (or updated) E2E cases (if required)
  - [ ] Passed E2E testing for **ALL** cases (local or/and sandbox)
- [ ] **Refactoring / Enhancement**
  - [ ] This is breaking change
  - [ ] Created (or updated) unit test cases (if required)
  - [ ] Passed unit testing for **ALL** cases
  - [ ] Created (or updated) E2E cases (if required)
  - [ ] Passed E2E testing for **ALL** cases (local or/and sandbox)
- [ ] **Bug-fix**
  - [ ] Type:
    - [ ] Code Issue:
    - [ ] Business Logic Issue:
  - [ ] Created (or updated) unit test cases (if required)
  - [ ] Passed unit testing for **ALL** cases
  - [ ] Created (or updated) E2E cases (if required)
  - [ ] Passed E2E testing for **ALL** cases (local or/and sandbox)

## Dependency

[Are there any required updates which cannot be done in this repo, or must be done after completing this PR?]

## Choose proper operation and replace the \<placeholder\>: `+semver:<placeholder>`

`breaking` `major` `feature` `minor` `fix` `patch`

+semver:feature

### CODE REVIEW

File: docs/SRS-Volere/SRS.tex [hunk 1]
```diff
@@ -108,13 +108,56 @@ \subsection{Assumptions}
...
+The current code plagiarism dtection tools such as MOSS rely on tokenization and syntax-level comparisons. Although effective
+in detecting direct copies or slight variations, they struggle when faced with techniques such as adding redundant code which
+allows the user to completely bypass detection while still plagiarizing the underlying logic and structure of the code.
+
+Additionally, MOSS does not take into account for the complexity or intent behind the code, leading to issues such as false
+positives for common programming patterns. This creates a gap for more advanced tools capable of understanding the semantic 
+meaning of code to more accurately detect plagiarism.
 \subsection{The Context of the Work}
-\lips
+Our prohject aims to address these gaps by incorporating Natural Language Processing (NLP) and machine learning techniques
```
Comment (lucasichen) [inline docs/SRS-Volere/SRS.tex:119]: Correct spelling of project

File: docs/SRS-Volere/SRS.tex [hunk 2]
```diff
@@ -108,13 +108,56 @@ \subsection{Assumptions}
...
+  \item \textbf{Documentation and Deployment}: Document the system architecture, the model, and the results. Deploy the system for use within academic settings.
+\end{itemize}
 \subsection{Specifying a Business Use Case (BUC)}
-\lips
+
+\textbf{Business Use Case:} Automated Code Plagiarism Detection for Academic Institutions
+
+\begin{itemize}
+    \item \textbf{Actors:} Professors, Students, System Administrators
+    \item \textbf{Trigger:} A professor or system administrator uploads multiple code submissions for plagiarism detection in a course assignment.
```
Comment (lucasichen) [inline docs/SRS-Volere/SRS.tex:143]: our current approach will only take into account one code submission vs another. It does not support multi code submission.

**Review by lucasichen — CHANGES_REQUESTED** [review]

**Comment by MohammadMohsinKhan** [comment]
> Change PR title to be more descriptive and when making commits please refer to the development plan on commit workflow. E.g. please use `feat: msg`, `fix: msg`, etc so we can differentiate between commit types.

"Change PR title to be more descriptive and when making commits please refer to the development plan on commit workflow. E.g. please use `feat: msg`, `fix: msg`, etc so we can differentiate between commit types." 🤓

File: docs/SRS-Volere/SRS.tex (hunk 2 above)
Comment (MohammadMohsinKhan) [reply docs/SRS-Volere/SRS.tex:143]: ACTUALLY, one code submission vs another is our POC plan which was outlined in the dev plan, I recommend you take a look at the document again to understand it. Our final product aims to do what I outlined.

File: docs/SRS-Volere/SRS.tex (hunk 1 above)
Comment (MohammadMohsinKhan) [reply docs/SRS-Volere/SRS.tex:119]: okay

**Review by lucasichen — APPROVED** [review]

**Pull request merged**